www/assets/en.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I think this should be in a subfolder of assets called something like translations.
Also, since we now have an assets folder, I think img should also be moved to the assets folder.